### PR TITLE
Fix test failures

### PR DESCRIPTION
**Pull request by 0xTim**
Fix parallel test failures introduced in #2998 - not sure how this got through CI ¯\_(ツ)_/¯

### CODE REVIEW

File: Tests/VaporTests/PipelineTests.swift [hunk 1]
```diff
@@ -86,7 +87,9 @@ final class PipelineTests: XCTestCase {
             }))
         }
         
-        try app.start()
+        app.environment.arguments = ["serve"]
```
Comment (gwynne) [inline Tests/VaporTests/PipelineTests.swift:90]: Why is this needed? Are we mishandling the default command or racing against other parallel tests that chance the commandline arguments?

**Review by gwynne — CHANGES_REQUESTED** [review]
See comment

File: Tests/VaporTests/PipelineTests.swift (hunk 1 above)
Comment (0xTim) [reply Tests/VaporTests/PipelineTests.swift:90]: It's always been like this - something about running the tests from CI seems to mishandle a default command
